lang/java/compiler/src/test/idl/output/forward_ref.avpr

```diff
@@ -8,7 +8,7 @@
       "fields": [
         { "name":"name", "type": "string", "doc":"the name" },
         { "name": "value", "type": "string", "doc": "the value" },
-        { "name": "type", "type": { "type": "enum", "name":"ValueType", "symbols": ["JSON","BASE64BIN","PLAIN"] } }
+        { "name": "type", "type": { "type": "enum", "name":"ValueType", "symbols": ["JSON","BASE64BIN","PLAIN"] }, "default": "PLAIN" }
```

Comment:
This is the default value specified in `forward_ref.avdl`. Its omission was a bug in the test case.